app/views/layouts/footer.html.erb

```diff
@@ -0,0 +1,16 @@
+<footer class="container-fluid footer">
+  <div class="row d-flex align-items-end">
+    <div class="footer__branding col-4 d-none d-md-block text-left footer__branding<%= 'footer__branding__login' if controller_name == 'home' && action_name == 'index' %>">
```

Comment:
This is more a nitpick, but I'm a if then logic in a helper and you can us the method here only for testability

Reply:
there yooo